File: cs/Iris/ParserUtils/LexParseException/InvalidRightTypeOperand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class InvalidRightTypeOperand : InvalidTypeOperand
    {
        public InvalidRightTypeOperand(int lNum, int cNum, Token.TokenValue token)
            : base(lNum, cNum, token)
        {
            
        }

        public override string ExceptionToString()
        {
            return base.ExceptionToString("right");
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/InvalidRightTypeOperand.java
package com.rokuan.iris.exception;

import com.rokuan.iris.parser.Token;

public class InvalidRightTypeOperand extends InvalidTypeOperand
{
	public InvalidRightTypeOperand(int lNum, int cNum, Token.TokenValue token)
	{
		super(lNum, cNum, token);            
	}

	public String ExceptionToString()
	{
		return super.ExceptionToString("right");
	}
}
